homeassistant/components/lametric/select.py

```diff
@@ -0,0 +1,91 @@
+"""Support for LaMetric selects."""
+from __future__ import annotations
+
+from collections.abc import Awaitable, Callable
+from dataclasses import dataclass
+from typing import Any
+
+from demetriek import BrightnessMode, Device, LaMetricDevice
+
+from homeassistant.components.select import SelectEntity, SelectEntityDescription
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity import EntityCategory
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import DOMAIN
+from .coordinator import LaMetricDataUpdateCoordinator
+from .entity import LaMetricEntity
+
+
+@dataclass
+class LaMetricEntityDescriptionMixin:
+    """Mixin values for LaMetric entities."""
+
+    options: list[str]
```

Comment:
Per https://github.com/home-assistant/core/pull/78882, should this be changed to something like `extended_options` (see similar situation w/ RainMachine: https://github.com/home-assistant/core/pull/79249)?

Reply:
That PR has not been approved.